lib/rspec/rails/configuration.rb

```diff
@@ -48,6 +48,15 @@ def self.initialize_configuration(config)
       config.include RSpec::Rails::ViewExampleGroup,       :type => :view
       config.include RSpec::Rails::FeatureExampleGroup,    :type => :feature
 
+      config.before :suite do
+        return unless defined?(ActiveRecord) && defined?(::RSpec::Mocks)
```

Comment:
Is the `return` being called locally?  Try changing it to `return if true` and I imagine it will fail in the same way.